Week3/homework/js-exercises/fortuneTeller.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+function tellFortune(number,partner,location,job){
+return  `In ${time[Math.floor(Math.random()*5)]}, You will be a ${job[Math.floor(Math.random()*5)]} in ${location[Math.floor(Math.random()*5)]}, and married to ${partner[Math.floor(Math.random()*5)]} with ${number[Math.floor(Math.random()*5)]} kids.`
```

Comment:
Ah nice, but does this still works if the arrays are bigger (or smaller than) 5? If there are more (or less) than 5 elements in it?

Reply:
no, it will not work, it will only run till first element.
should i make it work without *5?